debian/cuttlefish-base.cuttlefish-host-resources.default

```diff
@@ -1,4 +1,4 @@
-# defaults for cuttlefish-common
+# defaults for cuttlefish-base
```

Comment:
nit: should it cuttlefish-host-resources here?

Reply:
Done